enhance whiteboarding page with copy, paste and delete details

Nyae44 opened the pull request:
**Notes for Reviewers**

This PR fixes #460 

### Summary of changes
- Added instructions for copying, pasting and deleting shapes in Kanvas.

### How to Test 
1. Navigate to the whiteboarding page under **core tasks** [link](https://docs.layer5.io/kanvas/tasks/whiteboarding/). 
2. Verify that the section is well documented.
3. Check for consistency and correctness.

### Feedback Requested
- Are the instructions clear and easy to follow?




**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.
 

<!--
Thank you for contributing to Meshery! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
4. Build and test your changes before submitting a PR. 
5. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

leecalcote commented:
Thanks, @Nyae44!

miacycle commented:
cool!

vishalvivekm reviewed (changes requested):
@Nyae44 Do you mind letting me know, why is this deleted?

Nyae44 commented:
There was an extension in my IDE that generated some files each time I run the site on my local environment so I removed the extension and deleted the folder that it the extension had generated

miacycle commented on .gitignore:
Ideally, this file isn't changed...

miacycle commented:
Thanks for this, @Nyae44!
